qa-tests-backend/src/test/groovy/ReconciliationTest.groovy

```diff
@@ -154,7 +151,10 @@ class ReconciliationTest extends BaseSpecification {
                 log.info pod
             }
 
-            orchestrator.deleteAndWaitForDeploymentDeletion(sensorDeployment)
+            List<Pod> pods = orchestrator.getPodsByLabel("stackrox", ["app": "sensor"])
+            assert pods.size() == 1
+            orchestrator.scaleDeployment("stackrox", "sensor", 0)
+            orchestrator.deletePod("stackrox", pods[0].getMetadata().getName(), 0)
```

Comment:
I description you mentioned that scale is used instead of deletion but here we do both